.gitattributes

```diff
@@ -0,0 +1,3 @@
+$ cat .gitattributes
+* text=auto
+*.wren linguist-language=JavaScript
```

Comment:
what does this file do?

Reply:
enable hightlight in github for wren code. Since github does not highlight wren automatically I use JavaScript as an alternative until github properly detects wren code.